extra/update-emoji-picker-element.ts

```diff
@@ -0,0 +1,120 @@
+import * as fs from "fs";
+import { promisify } from "util";
+import * as path from "path";
+import { I18n } from "emoji-picker-element/shared";
+import de from "emoji-picker-element/i18n/de";
+import en from "emoji-picker-element/i18n/en";
+import es from "emoji-picker-element/i18n/es";
+import fr from "emoji-picker-element/i18n/fr";
+import it from "emoji-picker-element/i18n/it";
+import nl from "emoji-picker-element/i18n/nl";
+import pl from "emoji-picker-element/i18n/pl";
+import pt_PT from "emoji-picker-element/i18n/pt_PT";
+import ru_RU from "emoji-picker-element/i18n/ru_RU";
+
+const copyFile = promisify(fs.copyFile);
+const writeFile = promisify(fs.writeFile);
+const rm = promisify(fs.rm);
+const readdir = promisify(fs.readdir);
+
+if (process.argv.length !== 4) {
+  throw new Error(
+    "Expects the path to the directory in which the emoji data is saved as the #1 argument and the path to the Localisation.ts as the #2 argument.",
+  );
+}
+
+const repository = process.argv[2];
+if (!fs.existsSync(repository)) {
+  throw new Error(`The path '${repository}' does not exist.`);
+}
+
+const localisation = process.argv[3];
+if (!fs.existsSync(localisation)) {
+  throw new Error(`The path '${localisation}' does not exist.`);
+}
+
+const languages: LanguageItem[] = [
+  { local: "da" },
+  { local: "nl", i18n: nl },
+  { local: "en", i18n: en },
+  { local: "en-gb" },
+  { local: "et" },
+  { local: "fi" },
+  { local: "fr", i18n: fr },
+  { local: "de", i18n: de },
+  { local: "hu" },
+  { local: "it", i18n: it },
+  { local: "lt" },
+  { local: "nb" },
+  { local: "pl", i18n: pl },
+  { local: "pt", i18n: pt_PT },
+  { local: "ru", i18n: ru_RU },
+  { local: "es", i18n: es },
+  { local: "sv" },
+  { local: "uk" },
+];
+
+(async () => {
+  let localisationContent = `import { I18n } from "emoji-picker-element/shared";
```

Comment:
I strongly suggest adding a comment at the very top that states that this file is auto-generated and where to find the file to rebuild it. Make use of inline comments so that this is stripped from the bundled version.